src/main/java/uk/gov/register/resources/DataDownload.java

```diff
@@ -72,24 +72,25 @@ public RegisterSerialisationFormat downloadRSF() {
     }
 
     @GET
-    @Path("/download-rsf/{start-entry-no}/{end-entry-no}")
+    @Path("/download-rsf/{total-entries-1}/{total-entries-2}")
     @Produces({ExtraMediaType.APPLICATION_RSF, ExtraMediaType.TEXT_HTML})
     @DownloadNotAvailable
-    public RegisterSerialisationFormat downloadPartialRSF(@PathParam("start-entry-no") int startEntryNo, @PathParam("end-entry-no") int endEntryNo) {
-        if (startEntryNo < 1) {
-            throw new BadRequestException("start-entry-no must be greater than 0");
+    public RegisterSerialisationFormat downloadPartialRSF(@PathParam("total-entries-1") int totalEntries1, @PathParam("total-entries-2") int totalEntries2) {
+        if (totalEntries1 < 0) {
+            throw new BadRequestException("total-entries-1 must be positive");
```

Comment:
This error message is incorrect, because 0 is valid but it is not positive. Suggested alternatives:

 - "total-entries-1 cannot be negative"
 - "total-entries-1 must be nonnegative"
 - "total-entries-1 must be 0 or greater"